webapp/components/channel_invite_modal.jsx

```diff
@@ -30,8 +30,8 @@ export default class ChannelInviteModal extends React.Component {
         this.handleInviteError = this.handleInviteError.bind(this);
         this.search = this.search.bind(this);
 
-        this.page = 0;
         this.term = '';
+        this.page = 0;
```

Comment:
Shouldn't term and page be part of the state? I've missed most of these initial changes so there's probably something I'm missing here